### DISCUSSION TITLE
Since yesterday, all my builds are failing

### DISCUSSION THREAD

**Discussion by Filsommer** [vercel/community#2318] [Bug Report]
Preface: Other users are getting the same errors: https://www.reddit.com/r/nextjs/comments/138fl0v/strange_error_when_deploying_my_nextjs_project_to/

I can't deploy to Vercel by pushing code to main since yesterday. I get this strange error for every page on my website (404, homepage, etc.) even though my application doesn't contain a single useID() hook:

```
Error occurred prerendering page "/404". Read more: https://nextjs.org/docs/messages/prerender-error
TypeError: Cannot read properties of null (reading 'useId')
    at exports.useId (/vercel/path1/node_modules/react/cjs/react.production.min.js:24:350)
```
And I have no useId() hooks in my entire app... Has anyone else experienced this? What I have tried:
1. Rolling back to the last commit that worked
2. Deploying without package-lock.json
3. Updating NextJs and React to the latest version

Looks like the build error occurs when after:
```
info  - Compiled successfully
info  - Collecting page data...
info  - Generating static pages (0/10)  **<- After this line**
TypeError: Cannot read properties of null (reading 'useId')
```

Full error: [error.txt](https://github.com/vercel/community/files/11406232/error.txt)

**Comment by Filsommer** [comment]
Somehow it started working again.. Might have been an issue on their side.

**Reply by amyegan** [reply]
The team was already working on it, and a patch was released today: [v13.4.1](https://github.com/vercel/next.js/releases/tag/v13.4.1) 😄
